test/TestExpectations.json

```diff
@@ -269,6 +269,12 @@
     "parameters": ["webDriverBiDi"],
     "expectations": ["SKIP"]
   },
+  {
+    "testIdPattern": "[spec.spec] Page.pdf *",
```

Comment:
what is spec.spec? I think this might be skipping to much?

Reply:
This currently should do nothing... It should be `pdf.spec`, we don't support either `tagged` or `outlined` PDFs for BiDi.

Reply:
But we do support some other pdf methods? Please double check that we are not skipping more tests

Reply:
I will tomorrow, but this file only includes those 2 options.

Reply:
It skips only the two test as expected.